src/cloudant/account.py

```diff
@@ -440,16 +446,70 @@ def cors_origins(self):
         Retrieve a list of CORS origins
 
         """
-        pass
+        cors = self.cors_configuration()
+
+        return cors['origins']
 
     def update_cors_configuration(
             self,
             enable_cors=True,
             allow_credentials=True,
-            *origins):
+            origins=[],
```

Comment:
The other issue is that this refactor changes how the function works. The *origins at the end allows you to pass any number of values at the end of your argument list when calling this function, and have those values get rolled up into a tuple called "origins". With origins=[], you must pass a list, instead of being able to pass arbitrary args.

It would probably be best to go with "origins=None", and replace that None with a list in the logic in the body of the function.